media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A library to adjust the sound tempo, pitch and playback rate"
+DESCRIPTION="SoundTouch is an open-source audio processing library for \
+changing the tempo, pitch and playback rates of audio streams or audio files. \
+The library additionally supports estimating stable beats-per-minute rates for \
+audio tracks.
+
+- Tempo (time stretch): Changes the sound to play at faster or slower tempo \
+than originally without affecting the sound pitch.
+- Pitch (key) : Changes the sound pitch or key while keeping the original \
+tempo (speed).
+Playback Rate : Changes both tempo and pitch together as if a vinyl disc was \
+played at different RPM rate."
+HOMEPAGE="https://www.surina.net/soundtouch"
+COPYRIGHT="2001-2018 Olli Parviainen"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/soundtouch/soundtouch/-/archive/$portVersion/soundtouch-$portVersion.tar.bz2"
+CHECKSUM_SHA256="3fb740c7ac4b304342fbf11a91d14fa3e296269b9fec6266609c4620477d0ad6"
+SOURCE_DIR="soundtouch-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libsoundtouch$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:soundstretch$secondaryArchSuffix
+	lib:libSoundTouch$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libsoundtouch${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libSoundTouch$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+REQUIRES_devel="
+	libsoundtouch$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p config/m4
+	autoreconf -fi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	prepareInstalledDevelLibs libSoundTouch
+	fixPkgconfig
+
+	rm -Rf $prefix/doc
+
+	# devel package
+	packageEntries devel \
+		$developDir
```

Comment:
Suggestion:
```diff
-	rm -Rf $prefix/doc
+	rm -Rf "$prefix"/doc
 
 	# devel package
 	packageEntries devel \
-		$developDir
+		"$developDir"
 }
+
+TEST()
+{
+	make check
+}
```

Reply:
Yep, I know there are no tests, but there is a `check` target in the `Makefile`, so imho it is slightly better to have a `TEST()`, even if it _currently_ does nothing.

Reply:
Will add :)

Reply:
Is there any documentation in that "doc" folder? If so, I think we should package it.

Reply:
Good question. IDK.

Reply:
Removed, obselte :)